chroma-manager/tests/integration/shared_storage_configuration/test_alerting.py

```diff
@@ -151,7 +151,10 @@ def test_alerts(self):
                 self.assertEqual(alert_event['begin'], alert_event['end'])
 
             # Remove everything
-            self.graceful_teardown(self.chroma_manager)
+            self._fetch_help(lambda: self.graceful_teardown(self.chroma_manager),
+                             ['tom.nabarro@intel.com'],
+                             'failure in graceful teardown with unexpected volumes',
+                             timeout=999999)
```

Comment:
Or can we at least reduce the timeout to something smaller and more reasonable?

Reply:
have removed this one as it's no longer necessary